examples/relay-angular-chat/README.md

```diff
@@ -0,0 +1,27 @@
+# RelayAngularChat
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 13.2.2.
+
+## Development server
+
+Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
```

Comment:
For Markdown, best to use [semantic line breaks](https://sembr.org/). Makes editing easier, especially with git.